### PR TITLE
Add function help to Read The Docs as well as adding docstrings

### PR DESCRIPTION
**Pull request by texasaggie97**
[X] This contribution adheres to [CONTRIBUTING.md]
### What does this Pull Request accomplish?
* Add documentation to metadata for functions and function parameters
* Add docstrings to generated functions in session.py
* Add function reference on [readthedocs](http://nimi-python.readthedocs.io/en/docs-function_help/)

### Why should this Pull Request be merged?
* Updates documentation

### What testing has been done?
* Travis & System tests
* Visual inspection of documentation

### CODE REVIEW

File: docs/nidmm/functions.rst [hunk 1]
```diff
@@ -63,15 +63,15 @@ NI-DMM Functions
         :py:data:`nidmm.function` attribute to AC measurements. The driver sets the
         :py:data:`nidmm.AC\_MAX\_FREQ` attribute to this value. The valid range is 1
         Hz–300 kHz for the NI 4080/4081/4082 and the NI 4070/4071/4072, 10
-        Hz–100 Hz for the NI 4065, and 20 Hz–25 kHz for the NI 4050 and NI 4060.
+        Hz–100 Hz for the NI 4065, and 20 Hz–25 kHz for the NI 4050 and NI 4060.
```
Comment (marcoskirsch) [inline docs/nidmm/functions.rst]: any idea why GitHub thinks this line is different?

File: build/helper.py [hunk 2]
```diff
@@ -177,10 +177,16 @@ def get_method_parameters_snippet(parameters, skip_session_handle, skip_output_p
             snippets.append(x['python_name'])
     return ', '.join(snippets)
 
-def get_function_parameters_snippet(parameters):
-    '''Returns a string suitable for the parameter list of a method given a list of parameter objects'''
+def get_function_parameters_snippet(parameters, sessionName=None):
```
Comment (marcoskirsch) [inline build/helper.py]: should be session_name
Weird that default is different than in other functions. We should make it match or stop using default parameter value everywhere.

Comment (marcoskirsch) [inline build/helper.py]: In my last PR, I added skip_foo bool parameters. I think we should follow a consistent pattern.

File: src/nidmm/system_tests/test_system_nidmm.py [hunk 3]
```diff
@@ -146,7 +146,8 @@ def test_multi_point_acquisition(device_info):
         session.configure_multi_point(4, 2, nidmm.SampleTrigger.IMMEDIATE, 0)
         session.configure_measurement_digits(nidmm.Function.DC_VOLTS, 1, 5.5)
         measurements, numberOfMeasurements = session.read_multi_point(-1, 8)
-        print(measurements)
+        for measurement in measurements:
```
Comment (marcoskirsch) [inline src/nidmm/system_tests/test_system_nidmm.py]: Less important, but we should keep the PRs focused. This doesn't seem to be relevant to this PR.

File: docs/nidmm/functions.rst (hunk 1 above)
Comment (texasaggie97) [inline docs/nidmm/functions.rst]: To make it work with Python2, I change non-breaking spaces '\xa0' to normal spaces.

File: build/helper.py (hunk 2 above)
Comment (texasaggie97) [inline build/helper.py]: So to skip the session name you would need to parameters? skip_session_name=True and session_name="vi"?

Right now it is setup that only one parameter is needed: session_name.

File: src/nidmm/system_tests/test_system_nidmm.py (hunk 3 above)
Comment (texasaggie97) [inline src/nidmm/system_tests/test_system_nidmm.py]: I agree this isn't related to this PR. 

How should we deal with these type of small improvements though? It seems like a lot of overhead to try create a new branch, make a change like this, create a PR and then wait for it to be approved.
